controllers/kustomization_decryptor.go

```diff
@@ -406,13 +409,13 @@ func (d *KustomizeDecryptor) decryptKustomizationEnvSources(visited map[string]s
 				} else {
 					filePath = key
 				}
-				if err := visitRef(filePath, formats.FormatForPath(key)); err != nil {
+				if err := visitRef(filePath, formatForPath(key)); err != nil {
 					return err
 				}
 			}
 			for _, envFile := range gen.EnvSources {
-				format := formats.FormatForPath(envFile)
-				if formats.FormatForPath(envFile) == formats.Binary {
+				format := formatForPath(envFile)
+				if formatForPath(envFile) == formats.Binary {
```

Comment:
It looks like both the previous statement and the new statement both repeat the call to determine the format for the "envFile" variable.
```suggestion
				if format == formats.Binary {
```